pynestml/codegeneration/resources_nest/point_neuron/common/NeuronClass.jinja2

```diff
@@ -336,8 +360,12 @@ void {{neuronName}}::init_state_(const Node& proto)
 
 void {{neuronName}}::init_buffers_()
 {
-{%- for inputPort in neuron.get_input_ports() %}
-{{ buffer_initialization.BufferInitialization(inputPort) }}
+{% if (neuron.get_spike_input_ports())|length > 0 -%}
+  get_spike_inputs_().clear(); // includes resize
```

Comment:
what does "include resize" mean here?

Reply:
This was copied over from an old NESTML template code. I think it means that the `clear` function clears all the elements of the vector and sets the size to 0. I will remove it to avoid any confusion.

Reply:
Oh, I see, thanks for the explanation.